build_status.html

```diff
@@ -0,0 +1,353 @@
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="row row-copirayt">
+          <p class="mbr-text mb-0 mbr-fonts-style mbr-white align-center display-7">
+          © British Crown Copyright 2018, Met Office - All Rights Reserved
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- END FOOTER -->
+
+  <!-- START POST-INCLUDES -->
+  <!-- DO NOT MODIFY THIS BLOCK DIRECTLY. CHANGE index.html THEN RE-BUILD -->
+  <script src="/static/jquery/jquery.min.js">
+  </script>
+  <script src="/static/popper/popper.min.js">
+  </script>
+  <script src="/static/tether/tether.min.js">
+  </script>
+  <script src="/static/bootstrap/js/bootstrap.min.js">
+  </script>
+  <script src="/static/smoothscroll/smooth-scroll.js">
+  </script>
+  <script src="/static/touchswipe/jquery.touch-swipe.min.js">
+  </script>
+  <script src="/static/parallax/jarallax.min.js">
+  </script>
+  <script src="/static/sociallikes/social-likes.js">
+  </script>
+  <script src="/static/dropdown/js/script.min.js">
+  </script>
+  <script src="/static/theme/js/script.js">
+  </script>
+  <!-- END POST-INCLUDES -->
+
+            <script>
+              repos = [{org: 'SciTools', name: 'iris'},
+                       {org: 'SciTools', name: 'cartopy'},
+                       {org: 'SciTools', name: 'cf_units'},
+                       {org: 'SciTools', name: 'iris-grib'},
+                       {org: 'SciTools', name: 'nc-time-axis'},
+                       {org: 'SciTools', name: 'courses'},
+                       {org: 'SciTools-incubator', name: 'python-stratify'},
+                       {org: 'SciTools-incubator', name: 'iris-agg-regrid'},
+                      ];
+
+              table = $('#status-table');
+              body = $('tbody')
+
+              function build_row(repo) {
+                     return $('<tr>'
+                            + ' <td class="body-item mbr-fonts-style display-7">' + repo.name + '</td>'
+                            + ' <td class="body-item mbr-fonts-style display-7">'
+                            + '   <a href="https://coveralls.io/github/' + repo.org + '/' + repo.name + '?branch=master">'
+                            + '    <img src="https://travis-ci.org/' + repo.org + '/' + repo.name + '.svg?branch=master" alt="iris-build-status" title="iris-build-status">' 
+                            + '   </a>'
+                            + ' </td>'
+                            + ' <td class="body-item mbr-fonts-style display-7">'
+                            + '   <a href="https://coveralls.io/github/' + repo.org + '/' + repo.name + '?branch=master">'
+                            + '     <img src="https://coveralls.io/repos/github/' + repo.org + '/' + repo.name + '/badge.svg?branch=master" alt="Iris Coverage Status" />'
+                            + '   </a>'
+                            + ' </td>'
```

Comment:
Could you add the other badges like the ones at https://github.com/scitools/cf_units.